### PR TITLE
Feature/saves

### PR DESCRIPTION
**Pull request by MichelleCCodes**
# Description 
- alert only notifies that judge has been saved once 
- only saves judge to database once 
- successfully deletes judges
- Instead of sending judge_name to backend, it sends judge_id

## Type of Change
- [x] bug fix 
- [x] new feature

## Notes: 
 - There is a bug that occurs after trying to delete the judge. Error is coming from params. Need to inspect how MUI is updating. Code that introduces bug is commented out for now. The rest of the function works smoothly.

### CODE REVIEW

File: src/components/pages/JudgePage/JudgePage.js [hunk 1]
```diff
@@ -68,7 +68,6 @@ export default function JudgePage(props) {
           },
         })
         .then(res => {
-          console.log(res.data);
```
Comment (katieolson84) [inline src/components/pages/JudgePage/JudgePage.js:71]: Nice work cleaning up the console log

File: src/components/pages/SavedJudges/SavedJudges.js [hunk 2]
```diff
@@ -18,16 +18,16 @@ function SavedJudges({ savedJudges, deleteSavedJudge }) {
...
             className="savedJudgeLink"
-          >
-            <span>{params.value}</span>
-          </Link>
+          > */}
+          <span>{params.value}</span>
+          {/* </Link> */}
         </>
+        // ^^produces error after judge is deleted - bug
       ),
```
Comment (katieolson84) [inline src/components/pages/SavedJudges/SavedJudges.js:31]: I like that you added your own comment when you commented out the code so we know to keep it and look into it later.

File: src/components/pages/SavedJudges/SavedJudges.js [hunk 3]
```diff
@@ -84,7 +82,6 @@ function SavedJudges({ savedJudges, deleteSavedJudge }) {
   savedJudges.forEach((item, idx) => {
     item.id = idx;
   }); // this is VERY hacky, but the table doesn't take data without ids
```
Comment (katieolson84) [inline src/components/pages/SavedJudges/SavedJudges.js:84]: Note to discuss: This was already in the code from before, but I am wondering if we should rephrase this comment, maybe simply say "id needed for table data"

**Review by katieolson84 — APPROVED** [review]
You cleaned up a lot of the code and made a huge improvement to this feature to allow it to work. Approved, as there are no corrections needed. Keep up the great work.

File: src/components/pages/Home/RenderHomePage.js [hunk 4]
```diff
@@ -87,11 +87,7 @@ function RenderHomePage(props) {
   const [myPendingCases, setMyPendingCases] = useState([]);
   const user = useContext(UserContext);
 
-  // should move these API calls into a separate index folder at some point
-
   useEffect(() => {
-    // trackPromise(
-    // Tracks the axios call and implements spinning loader while executing
```
Comment (paulstgermain) [inline src/components/pages/Home/RenderHomePage.js:94]: Glad to see further cleaning up of our codebase here by removing old comments!

**Pull request merged**